openmetadata-ui/src/main/resources/ui/cypress/e2e/Features/DataConsumerRole.spec.js

```diff
@@ -253,7 +253,7 @@ describe('DataConsumer Edit policy should work properly', () => {
     }
     cy.get('body').click();
 
-    cy.get('[data-testid="add-new-glossary"]')
+    cy.get('[data-testid="add-glossary"]')
```

Comment:
We have both the buttons, Can you make sure that this change is required one?

Reply:
Yes, with this change we can run cypress for glossaries even if other glossary data already exist, earlier if other gossary data was there it user to fail the test